randovania/games/dread/json_data/header.txt

```diff
@@ -111,31 +111,222 @@ Templates
               Bomb and Water Bomb Jump (Beginner)
               Cross Bomb and Water Bomb Jump (Intermediate)
 
+* Open Charge Door:
+      Any of the following:
+          Shoot Charge Beam
+          Knowledge (Beginner) and Lay Power Bomb
+
+* Fight Silver Robot:
+      Any of the following:
+          Knowledge (Beginner) and Lay Power Bomb
+          All of the following:
+              Any of the following:
+                  # Methods of dodging
+                  Flash Shift or Combat (Beginner)
+              Any of the following:
+                  # Methods of fighting
+                  Shoot Plasma Beam
+                  Combat (Intermediate) and Shoot Charge Beam
+                  All of the following:
+                      # 47 Missiles are required, 1.5x leeway for trickless
+                      Shoot Missile
+                      Any of the following:
+                          Missiles ≥ 61
+                          Missiles ≥ 47 and Combat (Beginner)
+                  All of the following:
+                      # 28 Super Missiles are required, 1.5x leeway for trickless
+                      Shoot Super Missile
+                      Any of the following:
+                          Missiles ≥ 42
+                          Missiles ≥ 28 and Combat (Beginner)
+                  All of the following:
+                      # 15 Storm Missiles are required, 1.5x leeway for trickless
```

Comment:
might be better to indicate the number of lock-ons necessary in the comment rather than the raw ammo counts